### PR TITLE
vnstat2: update to version 2.11

### PR DESCRIPTION
**Pull request by janh**
Maintainer: me
Compile tested: x86/64, QEMU/KVM
Run tested: x86/64, QEMU/KVM, tested different configurations including update from legacy database

### CODE REVIEW

File: net/vnstat2/patches/100-noexit.patch [hunk 1]
```diff
@@ -58,3 +58,27 @@
...
++	int rc, i, dbifcount;
+ 	char sql[256];
+ 	const char *datatables[] = {"fiveminute", "hour", "day", "month", "year"};
+ 	const int32_t entrylimits[] = {cfg.fiveminutehours * 12, cfg.hourlydays * 24, cfg.dailydays, cfg.monthlymonths, cfg.yearlyyears};
+@@ -987,8 +987,11 @@ int db_removeoldentries(void)
+ 		printf("db: removing old entries\n");
+ 	}
+ 
+-	if (db_getiflist(&dbifl) <= 0) {
++	dbifcount = db_getiflist(&dbifl);
```
Comment (1715173329) [inline net/vnstat2/patches/100-noexit.patch:77]: please add a simple description for this change in your commit message:
why is this required?

Comment (janh) [reply net/vnstat2/patches/100-noexit.patch:77]: I extended the commit message to mention why this is necessary so the noexit patch keeps working.

Comment (1715173329) [reply net/vnstat2/patches/100-noexit.patch:77]: Thank you!
